### DISCUSSION TITLE
Can I uncomment?

### DISCUSSION THREAD

**Discussion by lxhandsome** [dbeaver/dbeaver#22196] [DBeaver Development]
![代码片段](https://github.com/lxhandsome/images/blob/main/clipbord_1702883544156.png)

Why do I need to comment this line? When I refresh, the newly added objects are not refreshed. They can be refreshed after I uncomment them.

**Comment by LonwoLonwo** [comment]
Hello @lxhandsome 

Please provide more context. It is problematic to understand your question right now.
